easybuild/easyconfigs/n/NetPIPE/NetPIPE-5.1-intel-2018a-mpi.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'MakeCp'
+
+name = 'NetPIPE'
+version = '5.1'
+variant = 'mpi'
+versionsuffix = '-%s' % variant
+
+homepage = 'http://netpipe.cs.ksu.edu/'
+description = """NetPIPE is a protocol independent communication performance benchmark that visually represents
+ the network performance under a variety of conditions."""
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+
+source_urls = ['http://netpipe.cs.ksu.edu/download/']
+sources = ['NetPIPE-%(version)s.tar.gz']
+checksums = ['0b6e6fd90446d06b70f0e615c56698887d3bb90b8a6be5d6d8a0ccb7efcd7b0b']
+
+# possible values: mpi, ibverbs, tcp, memcpy, disk, theo, shmem
+buildopts = variant
```

Comment:
@boegel - are you going to submit any of the other variants, too?

Reply:
Not planning to for now, I only wanted to play around with this particular one...

Maybe it's better if we make this a "fat" installations though, which includes all variants (and doesn't have a versionsuffix)?

Reply:
Yes, I think that would be more useful for people.